draft-ietf-quic-http.md

```diff
@@ -535,72 +526,46 @@ The PRIORITY frame payload has the following fields:
     {{prioritized-element-types}}). When sent on a request stream, this MUST be
     set to `11`.  When sent on the control stream, this MUST NOT be set to `11`.
 
-  DT (Element Dependency Type):
-  : A two-bit field indicating the type of element being depended on (see
-    {{element-dependency-types}}).
-
+  
   Empty:
-  : A four-bit field which MUST be zero when sent and MUST be ignored
+  : A six-bit field which MUST be zero when sent and MUST be ignored
     on receipt.
 
   Prioritized Element ID:
   : A variable-length integer that identifies the element being prioritized.
     Depending on the value of Prioritized Type, this contains the Stream ID of a
-    request stream, the Push ID of a promised resource, a Placeholder ID of a
-    placeholder, or is absent.
-
-  Element Dependency ID:
-  : A variable-length integer that identifies the element on which a dependency
-    is being expressed. Depending on the value of Dependency Type, this contains
-    the Stream ID of a request stream, the Push ID of a promised resource, the
-    Placeholder ID of a placeholder, or is absent.  For details of
-    dependencies, see {{priority}} and {{!RFC7540}}, Section 5.3.
-
-  Weight:
-  : An unsigned 8-bit integer representing a priority weight for the prioritized
-    element (see {{!RFC7540}}, Section 5.3). Add one to the value to obtain a
-    weight between 1 and 256.
-
-The values for the Prioritized Element Type ({{prioritized-element-types}}) and
-Element Dependency Type ({{element-dependency-types}}) imply the interpretation
-of the associated Element ID fields.
+    request stream, the Push ID of a promised resource, or is absent.
+
+  Priority:
+  : An unsigned variable-length integer representing a priority for the prioritized
+    element (see {{!RFC7540}}, Section 5.3). It should be interpreted as a fixed point
```

Comment:
These were the lines that look odd for me and are slightly too long due to the two space indent.